utils/run_test.sh

```diff
@@ -68,6 +68,15 @@ run_test()
     fi
 
     ((log_num += 1))
+
+    FILES=${DAOS_BASE}/test_results/*
+    for f in $FILES
+    do
+        SUITE=`grep "testsuite name=" $f | grep -Po "name=\"\K.*(?=\" time=)"`
+        sed -i "s/testcase name=/testcase classname=\"${COMPONENT}.${SUITE}\" \
+        name=/" $f
+    done
+    mv ${DAOS_BASE}/test_results/*.xml ${DAOS_BASE}/test_results/xml
```

Comment:
(lint) Double quote to prevent globbing and word splitting. [SC2086]
(lint) Double quote to prevent globbing and word splitting. [SC2086]